src/server/debug.rs

```diff
@@ -1486,30 +1477,25 @@ mod tests {
     }
 
     impl Debugger<RocksEngine> {
-        fn get_store_ident(&self) -> Result<StoreIdent> {
+        fn set_store_id(&self, store_id: u64) {
+            let mut ident = self.get_store_ident().unwrap_or_default();
+            ident.set_store_id(store_id);
             let db = &self.engines.kv;
-            db.get_msg::<StoreIdent>(keys::STORE_IDENT_KEY)
-                .map_err(|e| box_err!(e))
-                .map(|ident| match ident {
-                    Some(ident) => ident,
-                    None => StoreIdent::default(),
-                })
+            db.put_msg(keys::STORE_IDENT_KEY, &ident).unwrap();
         }
 
-        fn set_store_id(&self, store_id: u64) {
-            if let Ok(mut ident) = self.get_store_ident() {
-                ident.set_store_id(store_id);
-                let db = &self.engines.kv;
-                db.put_msg(keys::STORE_IDENT_KEY, &ident).unwrap();
-            }
+        fn set_cluster_id(&self, cluster_id: u64) {
+            let mut ident = self.get_store_ident().unwrap_or_default();
```

Comment:
`unwrap_or_default` is not compatible with the origin logic. Will it be a problem?

Reply:
@zhongzc Not really. This is the original logic
![image](https://user-images.githubusercontent.com/15794564/141080148-0cd2f6c3-b16b-4754-91d1-c798e019b446.png)

Reply:
@yuqi1129 

original (pseudocode):
```
match db.get_msg:
    case Ok(Some(id)) => id
    case Ok(None) => StoreIdent::default()
    case Err(_) => noop
```

here (pseudocode):
```
match db.get_msg:
    case Ok(Some(id)) => id
    case Ok(None) => StoreIdent::default()
    case Err(_) => StoreIdent::default()
```

Reply:
Well, I just found out that it's just test code. Not a big problem.